financial-connections/src/main/java/com/stripe/android/financialconnections/domain/NativeAuthFlowRouter.kt

```diff
@@ -19,25 +19,18 @@ internal class NativeAuthFlowRouter @Inject constructor(
 
     fun nativeAuthFlowEnabled(
         manifest: FinancialConnectionsSessionManifest,
-        isInstantDebits: Boolean
     ): Boolean {
         debugConfiguration.overriddenNative?.let { return it }
         val killSwitchEnabled = nativeKillSwitchActive(manifest)
-        val nativeExperimentEnabled = manifest
-            .experimentAssignment(Experiment.CONNECTIONS_MOBILE_NATIVE) == EXPERIMENT_VALUE_NATIVE_TREATMENT
-        return isInstantDebits.not() && killSwitchEnabled.not() && nativeExperimentEnabled
+        return killSwitchEnabled.not() && nativeExperienceEnabled(manifest)
```

Comment:
The removal of `isInstantDebits.not()` does most of the heavy lifting of this pull request.